python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -576,12 +576,18 @@ def makeGraph(self, pipeline: Pipeline, args: SimpleNamespace) -> Optional[Quant
         if nQuanta == 0:
             return None
         else:
-            _LOG.info(
-                "QuantumGraph contains %d quanta for %d tasks, graph ID: %r",
-                nQuanta,
-                len(qgraph.taskGraph),
-                qgraph.graphID,
+            qg_tasks = []
```

Comment:
Since iterating through the graph is not "free" can you add a `if _LOG.isEnabledFor(logging.INFO)`  around this?